syntax = "proto2";

package mediapipe;

option java_package = "com.google.mediapipe.formats.proto";
option java_outer_classname = "CaratFaceEffectProto";

message CaratFaceEffect {
    optional string texture_path = 1;
    optional string mesh_3d_path = 2;
}

message CaratFaceEffectSet {
    optional int64 id = 1;
    repeated CaratFaceEffect caratFaceEffect = 2;
}